tests/containers/buildah_image.pm

```diff
@@ -26,23 +26,23 @@ use version_utils qw(get_os_release check_os_release);
 sub run {
     my ($image_names, $stable_names) = get_suse_container_urls();
     my ($running_version, $sp, $host_distri) = get_os_release;
-    my $runtime = "buildah";
 
     install_buildah_when_needed($host_distri);
-    allow_selected_insecure_registries(runtime => $runtime);
+    install_podman_when_needed($host_distri);
+    allow_selected_insecure_registries(runtime => 'podman');
     scc_apply_docker_image_credentials() if (get_var('SCC_DOCKER_IMAGE'));
 
     for my $iname (@{$image_names}) {
-        test_container_image(image => $iname, runtime => $runtime);
+        test_container_image(image => $iname, runtime => 'buildah');
         if (check_os_release('suse', 'PRETTY_NAME')) {
             # sle15-working-container is the default name given to a container. it is created in test_container_image
-            test_opensuse_based_image(image => 'sle15-working-container', runtime => $runtime);
+            test_opensuse_based_image(image => 'sle15-working-container', runtime => 'buildah');
             allow_selected_insecure_registries(runtime => 'podman');
-            test_containered_app('podman', $iname);
         }
     }
     scc_restore_docker_image_credentials();
     clean_container_host(runtime => 'podman');
```

Comment:
Is this still requiredin `buildah_image.pm`?

Reply:
Readded the comment, because it got accidentally deleted.

Reply:
Yes, in this module both `buildah` and `podman` produce images.